### ISSUE TITLE
Recurrent implementation

### ISSUE THREAD

**Issue by mishmetall** [rinuboney/ladder#10]
It was mentioned in the original article that such kind of model could be easily used in RNNs. I'm trying to implement such a model now, but I wonder which of the connections should be batch-normilized. 
This paper (https://arxiv.org/pdf/1603.09025v4.pdf) provides instructions to use several batch normalizations across LSTM calculation routine, another (https://arxiv.org/pdf/1510.01378v1.pdf), and also this approach works for my data, not the first, say that we should normalize only inputs. Are there any ideas and research how effectively transform LSTM into Ladder LSTM?

**Comment by attaullah** [comment]
@mishmetall  have you tried ladder for CIFAR-10?

**Comment by Felaadar** [comment]
@attaullah no, I haven't, I've tried ladder on my own datasets, results were contradictory (in some cases I saw improvements, in some not, overall more research needed, but no breakthroughs in performance in compare with pure supervised learning)
